src/Map/src/Bridge/Google/assets/package.json

```diff
@@ -40,6 +40,7 @@
     "devDependencies": {
         "@googlemaps/js-api-loader": "^1.16.6",
         "@hotwired/stimulus": "^3.0.0",
+        "@symfony/ux-map": "workspace:*",
```

Comment:
It has no impact on `dist/` files, either used by Webpack Encore or AssetMapper. Here we are telling Yarn that package `@symfony/ux-map` is a "parent" package of `@symfony/ux-google-map`, in order to fix the topological order (see https://yarnpkg.com/cli/workspaces/foreach)